packages/frontend-app/src/app/components/CookieBanner.tsx

```diff
@@ -1,13 +1,25 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import classNames from "classnames";
 
 import { Button } from "./Button";
 import useCookieConsent from "../hooks/useCookieConsent";
 
 export const CookieBanner = () => {
   const [value, consentToCookie] = useCookieConsent();
+  const [isVisible, setIsVisible] = useState(false);
```

Comment:
```suggestion
  const [isVisible, setIsVisible] = useState(!!value);
```

Doing this is better than having a useEffect just to set the visible state. Also the component does not go through any unnecessary re-render

Reply:
Changes have been applied accordingly. The approach reduces the number of renders and makes the code cleaner by handling the initial visibility directly within the useState initialization.